kubevirt-disk-uploader.yaml

```diff
@@ -49,8 +49,12 @@ spec:
             secretKeyRef:
               name: kubevirt-disk-uploader-credentials
               key: registryPassword
+        - name: VM_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
       command: ["/usr/local/bin/kubevirt-disk-uploader"]
-      # args: ["--vmname", "example-vm", "--volumename", "datavolumedisk", "--imagedestination", "quay.io/boukhano/example-vm-exported:latest", "--enablevirtsysprep", "false", "--pushtimeout", "120"]
+      # args: ["--vmnamespace", "default", "--vmname", "example-vm", "--volumename", "datavolumedisk", "--imagedestination", "quay.io/boukhano/example-vm-exported:latest", "--enablevirtsysprep", "false", "--pushtimeout", "120"]
```

Comment:
in this example, I would not set any hardcoded namespace and just keep the value to be set by k8s.